docs: add airgap VMO profile creation PEM-5497 (#3583)
spectrocloud/librarium: docs/docs-content/vm-management/create-vmo-profile.md

</TabItem>

<TabItem value="airgap" label="Airgap">

> 1. Log in to Palette using your configured system address.

Reviewer: ```suggestion
1. Log in to a tenant that belongs to your instance of Palette or Palette VerteX.
```

Reviewer: I feel like this is more specific, since system credentials could confuse people into thinking they should log in to the system console :eye:

> 2. Select **Profiles** in the left **Main Menu** and click the **Add Cluster Profile** button.

Reviewer: ```suggestion
2. In the **left Main Menu**, select **Profiles** and click **Add Cluster Profile**.
```

3. Enter basic information for the profile: name, version if desired, and optional description.

4. Select type **Add-on**, and click **Next**.

5. In the following screen, click **Add New Pack**.

6. Locate the **Virtual Machine Orchestrator** pack and add it to your profile.

> 7. Review the **Access** configuration panel at right. The default setting is **Proxied**, which automatically adds the

Reviewer: ```suggestion
7. Review the **Access** configuration panel on the right. The default setting is **Proxied**, which automatically adds the
```

   **Spectro Proxy** pack when you create the cluster, allowing access to the Spectro VM Dashboard from anywhere. Check
   out the <VersionedLink text="Spectro Proxy" url="/integrations/packs/?pack=spectro-proxy" /> guide to learn more.

>    Changing the default may require some additional configuration.

Reviewer: ```suggestion
```

   The **Direct** option is intended for a private configuration where a proxy is not implemented or not desired.

   :::warning

   We recommend using the pack defaults. Default settings provide best practices for your clusters. Changing the default
   settings can introduce misconfigurations. Carefully review the changes you make to a pack.

   :::

> 8. Click **Values** in the **Pack Details** section. The pack manifest editor appears.

Reviewer: ```suggestion
8. If you want to use the **Direct** access mode, click **Values** in the **Pack Details** section. The pack manifest editor appears.
```

> 9. Locate the `pack.cdi.privateRegistry` section in the manifest. The table below contains a brief description of each

Reviewer: ```suggestion
 Locate the `pack.cdi.privateRegistry` section in the manifest. The table below contains a brief description of each
```

   field exposed by the private registry. Set the `pack.cdi.privateRegistry.enabled` field to true and fill in the
   registry IP address and base path according to your environment. This configures the VMO pack to pull images from
   your airgap environment private registry.

   | Field                                       | Description                                                          |
   | ------------------------------------------- | -------------------------------------------------------------------- |
   | `pack.cdi.privateRegistry.enabled`          | Flag to enable the profile to use the airgap private image registry. |
   | `pack.cdi.privateRegistry.registryIP`       | The IP address the private image registry.                           |
   | `pack.cdi.privateRegistry.registryBasePath` | The base path of the private image registry.                         |

   ```yaml
   cdi:
     privateRegistry:
       enabled: true
       registryIP: <REPLACE ME>
       registryBasePath: <REPLACE ME>

>    ```

Reviewer: I would combine this all in one step, as in:

_If you want to use Direct access, open the yaml config, and then make these changes..._

It's a larger step but it would be more straightforward this way I think 🤔

10. Click **Confirm & Create**.

> 11. In the following screen, click **Next**.

Reviewer: ```suggestion
11. On the following screen, click **Next**.
```
